### PR TITLE
#31 product page featured product section

### PR DESCRIPTION
**Pull request by camargov**
I had issues viewing the text for the individual products on the product page, but I still styled the text so once the text is visible, less editing is needed and I was also wondering if the Featured Product section looks good enough.

### CODE REVIEW

File: src/components/products/products-styles.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
+import styled from "styled-components"
+import dimensions from "../../style/dimensions"
+
+export const ProductPageHeading = styled.div`
+  font-size: calc(90px + (90 - 32) * ((100vw - 300px) / (1440 - 300)));
```
Comment (samgildea) [inline src/components/products/products-styles.js]: Now that we have a typography file we should just import H1 from the typography-styles.js

File: src/components/products/products-styles.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  padding-top: 52px;
+
+  @media (max-width: ${dimensions.maxwidthTablet}px) {
+    padding-left: 0vw;
+    padding-top: 26px;
+  }
+`
+
+export const FeaturedProductHeading = styled.div`
+  font-size: calc(20px + (20 - 16) * ((100vw - 300px) / (1440 - 300)));
```
Comment (samgildea) [inline src/components/products/products-styles.js]: Same here,  use predefined typography styles

**Review by samgildea — CHANGES_REQUESTED** [review]

File: src/components/product_page/product-page.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+export default function ProductSection({ data }) {
+  return (
+    <div>
+      <ProductsHeading>
+        <H1>{data.product_page_main_heading}</H1>
+      </ProductsHeading>
+      <FeaturedProductSection>
+        <FeaturedProductImage>
+          {console.log(data.featured_product_image)}
```
Comment (josephmannis) [inline src/components/product_page/product-page.js]: Can you remove this please

File: src/components/product_page/product-page.js [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+            src={data.featured_product_image.url}
+            alt={data.featured_product_image.alt}
+          />
+        </FeaturedProductImage>
+        <FeaturedProductText>
+          <Sub1>{data.featured_product_heading}</Sub1>
+          <H2>{data.featured_product_name}</H2>
+          <Body>{data.featured_product_description}</Body>
+          <FeaturedCTA>
+            {console.log(data)}
```
Comment (josephmannis) [inline src/components/product_page/product-page.js]: This too

File: src/components/product_page/product-page.js [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+        </FeaturedProductText>
+      </FeaturedProductSection>
+
+      <HomeownerSection>
+        <Sub1>{data.model_scope[0].model_scope_heading}</Sub1>
+        <ProductsGroup>
+          {data.products.map(product => {
+            return (
+              product.product.document.data.product_type === "Homeowner" && (
+                <ProductCard href={`/${product.product.document.uid}`}>
```
Comment (josephmannis) [inline src/components/product_page/product-page.js]: Need a key

File: src/components/product_page/product-page.js [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+              )
+            )
+          })}
+        </ProductsGroup>
+      </HomeownerSection>
+
+      <CommercialSection>
+        <Sub1>{data.model_scope[1].model_scope_heading}</Sub1>
+        <ProductsGroup>
+          {console.log(data.products)}
```
Comment (josephmannis) [inline src/components/product_page/product-page.js]: Remove please

File: src/components/product_page/product-page.js [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+          {data.products.map(product => {
+            return (
+              product.product.document.data.product_type === "Commercial" && (
+                <ProductCard href={`/${product.product.document.uid}`}>
```
Comment (josephmannis) [inline src/components/product_page/product-page.js]: Need a key

**Review by josephmannis — CHANGES_REQUESTED** [review]
Little stuff, looks good otherwise!

**Pull request merged**